Makefile

```
@@ -196,11 +197,15 @@ bundle-image-build: bundle
 bundle-image-push:
 	$(CONTAINER_ENGINE) push $(BUNDLE_IMG)
 
+generate-catalog: opm ## Generate catalog for the Konflux-built operator
+	mkdir -p $(CATALOG_VERSION_DIR)
+	$(OPM) alpha render-template basic -o yaml $(CATALOG_DIR)/catalog-template.yaml > $(CATALOG_VERSION_DIR)/catalog.yaml
```

To get not base64 encoded CSV ([similar ALBO change](https://github.com/openshift/aws-load-balancer-operator/commit/e31851fe67b98a143fa3a1200c9bf1011369079a)):
```suggestion
	$(OPM) alpha render-template basic --migrate-level bundle-object-to-csv-metadata -o yaml $(CATALOG_DIR)/catalog-template.yaml > $(CATALOG_VERSION_DIR)/catalog.yaml
```